### PR TITLE
Add basic write load generator.

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

File: pkg/commands/loadgen.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+	"context"
+	"fmt"
+	"math/rand"
+	"net/url"
+	"sync"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/snappy"
+	"github.com/google/martian/log"
```
Comment (gotjosh) [inline pkg/commands/loadgen.go:13]:
```suggestion
	log "github.com/sirupsen/logrus"
```

Comment (gotjosh) [reply pkg/commands/loadgen.go:13]: This and a `go mod tidy` should fix the linter/test issues.

File: pkg/commands/loadgen.go [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+	r := v1.Range{
+		Start: time.Now().Add(-c.queryDuration),
+		End:   time.Now(),
+		Step:  time.Minute,
+	}
+	start := time.Now()
+	_, _, err := c.queryClient.QueryRange(ctx, c.query, r)
+	if err != nil {
+		queryRequestDuration.WithLabelValues("error").Observe(time.Now().Sub(start).Seconds())
+		log.Printf("error doing query: %v", err)
```
Comment (jtlisi) [inline pkg/commands/loadgen.go:211]:
```suggestion
		log.Printf("error doing query: %v", err)
		return
```

I think we need to return here or else the success metric will be logged as well

File: pkg/commands/loadgen.go [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+		req = prompb.WriteRequest{
+			Timeseries: make([]prompb.TimeSeries, 0, to-from),
+		}
+		now = time.Now().UnixNano() / int64(time.Millisecond)
+	)
+
+	for i := from; i < to; i++ {
+		timeseries := prompb.TimeSeries{
+			Labels: []prompb.Label{
+				{Name: "__name__", Value: "node_cpu_seconds_total"},
```
Comment (jtlisi) [inline pkg/commands/loadgen.go:161]: Might be better off choosing a metric name that doesn't already exist or make it configurable similar to the `-query` flag above. It's NBD to leave it as is. It's easy enough to change later.

Comment (tomwilkie) [reply pkg/commands/loadgen.go:161]: Good idea but yeah, lets fix later.

**Pull request merged**